Rakefile

```
@@ -13,11 +14,75 @@ namespace :db do
 end
 
 namespace :web do
+  desc 'Create test kit pages from test kit metadata'
+  task :create_test_kit_pages do
+
+    TEST_KIT_DIR = 'web/_test_kits'
+    require 'inferno'
+    ENV['NO_DB'] = 'true'
+    Inferno::Application.start(:suites)
+
+    FileUtils.rm_rf(Dir.glob("#{TEST_KIT_DIR}/*.md"))
+
+    config = YAML.safe_load(File.read('web/_config.yml'))
+    excluded_test_kits = config.fetch('excluded_test_kits', [])
+
+    test_kits = Inferno::Repositories::TestKits.new.all
+
+    test_kits.each do |test_kit|
+      next if excluded_test_kits.include?(test_kit.id.to_s)
+
+      formatted_id = test_kit.id.to_s.gsub('_', '-').sub(/-?test-kit$/, '')
```

```suggestion
      formatted_id = test_kit.id.to_s.dasherize.delete_suffix('-test-kit')
```
When wouldn't there be a `-` in front of `test-kit`?